models/model_upload/multimodal_models/vllm-miniCPM-2.6/1/model.py

```diff
@@ -0,0 +1,304 @@
+import base64
+import os
+import subprocess
+import sys
+import threading
+from typing import Iterator
+
+from clarifai.runners.models.model_runner import ModelRunner
+from clarifai.utils.logging import logger
+from clarifai_grpc.grpc.api import resources_pb2, service_pb2
+from clarifai_grpc.grpc.api.status import status_code_pb2
+from google.protobuf import json_format
+from openai import OpenAI
```

Comment:
Don't you need this in requirements.txt?

Reply:
Openai client installed with vLLM dependency

Reply:
nevermind, it is in the requirements.txt file anyway; I was looking at the wrong file